packages/core/package.json

```diff
@@ -1,13 +1,20 @@
 {
-  "name": "@loopback/loopback",
+  "name": "@loopback/core",
   "version": "4.0.0-alpha.1",
   "description": "",
   "main": "index",
   "scripts": {
     "acceptance": "mocha --opts ../../test/mocha.opts 'test/acceptance/**/*.ts'",
+    "build": "npm run build:lib && npm run build:lib6",
+    "build:lib": "tsc --target es2017 --outDir lib",
+    "build:lib6": "tsc --target es2015 --outDir lib6",
+    "clean": "rm *.tgz; rm -rf ./package",
+    "prepublish": "npm run build",
+    "pretest": "npm run build",
     "integration": "mocha --opts ../../test/mocha.opts 'test/integration/**/*.ts'",
     "test": "mocha --opts ../../test/mocha.opts 'test/unit/**/*.ts' 'test/integration/**/*.ts' 'test/acceptance/**/*.ts'",
-    "unit": "mocha --opts ../../test/mocha.opts 'test/unit/**/*.ts'"
+    "unit": "mocha --opts ../../test/mocha.opts 'test/unit/**/*.ts'",
+    "verify": "npm run clean && npm pack && tar xvzf *.tgz && tree package"
```

Comment:
Just tried it with `tar tf *.tgz`, but it's much harder to read/verify by eye than tree.

`tar xf *.tgz && tree package`
```
package
├── README.md
├── index.d.ts
├── index.js
├── lib
│   ├── binding.d.ts
│   ├── binding.js
│   ├── context.d.ts
│   ├── context.js
│   ├── index.d.ts
│   ├── index.js
│   ├── inject.d.ts
│   ├── inject.js
│   ├── isPromise.d.ts
│   ├── isPromise.js
│   ├── resolver.d.ts
│   └── resolver.js
├── lib6
│   ├── binding.d.ts
│   ├── binding.js
│   ├── context.d.ts
│   ├── context.js
│   ├── index.d.ts
│   ├── index.js
│   ├── inject.d.ts
│   ├── inject.js
│   ├── isPromise.d.ts
│   ├── isPromise.js
│   ├── resolver.d.ts
│   └── resolver.js
└── package.json
```

vs

`tar tf *.tgz`
```
package/package.json
package/README.md
package/index.js
package/lib/resolver.js
package/lib/context.js
package/lib/index.js
package/lib/binding.js
package/lib/inject.js
package/lib/isPromise.js
package/lib/binding.d.ts
package/lib/index.d.ts
package/lib/isPromise.d.ts
package/lib/context.d.ts
package/lib/resolver.d.ts
package/lib/inject.d.ts
package/index.d.ts
package/lib6/resolver.js
package/lib6/context.js
package/lib6/index.js
package/lib6/binding.js
package/lib6/inject.js
package/lib6/isPromise.js
package/lib6/binding.d.ts
package/lib6/index.d.ts
package/lib6/isPromise.d.ts
package/lib6/context.d.ts
package/lib6/resolver.d.ts
package/lib6/inject.d.ts
```